collector/nodes.go

```diff
@@ -1510,6 +1510,54 @@ func NewNodes(logger log.Logger, client *http.Client, url *url.URL, all bool, no
 				},
 				Labels: defaultNodeLabelValues,
 			},
+			{
+				Type: prometheus.CounterValue,
+				Desc: prometheus.NewDesc(
+					prometheus.BuildFQName(namespace, "ingest", "total_count"),
+					"Total number of ingest count",
+					defaultNodeLabels, nil,
+				),
+				Value: func(node NodeStatsNodeResponse) float64 {
+					return float64(node.Ingest.Total.Count)
+				},
+				Labels: defaultNodeLabelValues,
+			},
+			{
+				Type: prometheus.CounterValue,
+				Desc: prometheus.NewDesc(
+					prometheus.BuildFQName(namespace, "ingest", "total_time_in_millis"),
+					"Total number of ingest time in millis",
+					defaultNodeLabels, nil,
+				),
+				Value: func(node NodeStatsNodeResponse) float64 {
+					return float64(node.Ingest.Total.TimeInMillis)
+				},
+				Labels: defaultNodeLabelValues,
+			},
+			{
+				Type: prometheus.CounterValue,
+				Desc: prometheus.NewDesc(
+					prometheus.BuildFQName(namespace, "ingest", "total_current"),
```

Comment:
Is this a counter that always increases? If so `_total` should be at the end
```suggestion
					prometheus.BuildFQName(namespace, "ingest", "current"),
```

Reply:
In fact, it should be a guage because the data will be zeroed out due to JVM restart. i will fix them.

Reply:
If the type is Gauge, The "total" prefix should be appropriate？It means the sum of all pipeline stats on a node